pinot-broker/src/main/java/com/linkedin/pinot/broker/api/resources/PinotBrokerHealthCheck.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.broker.api.resources;
+
+import com.linkedin.pinot.broker.broker.BrokerServerBuilder;
+import com.linkedin.pinot.common.metrics.BrokerMeter;
+import com.linkedin.pinot.common.metrics.BrokerMetrics;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Api(tags = "Health")
```

Comment:
Do we want to do this via web API? If not, you can omit this, and it won't show up in swagger. I don't think we have swagger for brokers now, do we ?

Reply:
I set up the swagger for broker with this migration. Since server also has swagger api, I think that there's no harm to keep swagger. How do you think? Broker has total 3 apis (routing table, time boundary service and query)

Reply:
i like having swagger